### PR TITLE
apache-kafka: New package and service

### PR DESCRIPTION
**Pull request by ragnard**
New expressions for the Apache Kafka distributed messaging system. Provides both a new package 'apache-kafka', and a nixos service with the same name.

### CODE REVIEW

File: pkgs/servers/apache-kafka/default.nix [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  version = "${scalaVersion}-${kafkaVersion}";
+
+  src = fetchurl {
+    url = "mirror://apache/kafka/${kafkaVersion}/kafka_${version}.tgz";
+    sha256 = "1bya4qs0ccrqibmdivgdxcsyiay4c3vywddrkci1dz9v3ymrqby9";
+  };
+
+  buildInputs = [ jre makeWrapper bash ];
+
+  phases = ["unpackPhase" "installPhase" "fixupPhase"];
```
Comment (lethalman) [inline pkgs/servers/apache-kafka/default.nix]: You don't need this.

File: pkgs/servers/apache-kafka/default.nix [hunk 2]
```diff
@@ -0,0 +1,51 @@
+{ stdenv, fetchurl, jre, makeWrapper, bash }:
+
+let
+  kafkaVersion = "0.8.1.1";
+  scalaVersion = "2.8.0";
+
+in
+
+stdenv.mkDerivation rec {
+  name = "apache-kafka";
```
Comment (lethalman) [inline pkgs/servers/apache-kafka/default.nix]: Put a version in the name please.

**Comment by ragnard** [comment]
@lethalman Thanks! Updated PR with your comments

File: pkgs/servers/apache-kafka/default.nix [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        --set KAFKA_LOG_DIR "/tmp/apache-kafka-logs" \
+        --prefix PATH : "${bash}/bin"
+    done
+    chmod +x $out/bin\/*
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = "http://kafka.apache.org";
+    description = "Apache Kafka";
+    longDescription= "A high-throughput distributed messaging system";
```
Comment (pSub) [inline pkgs/servers/apache-kafka/default.nix]: Please use what you have as `longDescription` as `description` and remove `longDescription`. See for details http://nixos.org/nixpkgs/manual/#idm140737319319584

**Comment by ragnard** [comment]
@pSub Updated PR as requested. Thanks!

**Comment by lethalman** [comment]
Ok thanks.

**Comment by ragnard** [comment]
Thanks for reviewing and merging!

**Pull request merged**